### PR TITLE
add git workflow documentation 

### PR DESCRIPTION
**Pull request by ecle20yy**
This is a guideline for all squads to follow regarding the methods of working using git branches.

### CODE REVIEW

**Comment by ecle20yy** [comment]
@AlexTelon

**Comment by Lillenooben** [comment]
I'll leave this one for Alex to read through so you two and discuss incase there is something he wants changed, or have any suggestions.

File: GitPraxis.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+### Naming feature branches
+
+To make it really clear what issue is related to what branch, you shall name your feature branches according to the following template:
+
+```txt
+x/y-z
+
+x = 'feature' or 'bug' or 'hotfix'
+y = GitHub issue number (without #)
```
Comment (AlexTelon) [inline GitPraxis.md:29]: You should have the # so that in the github system you get links to issues.

Comment (AlexTelon) [reply GitPraxis.md:29]: I'm pretty sure you can then see the commits related to an issue from this as well.

**Review by AlexTelon — COMMENTED** [review]
Other than my one comment I think this is a good starting point

File: GitPraxis.md (hunk 1 above)
Comment (ecle20yy) [reply GitPraxis.md]: Edited.

**Comment by ecle20yy** [comment]
@AlexTelon Now, that should be correct. I've added a # to the example feature branch.

**Review by Lillenooben — APPROVED** [review]

**Pull request merged**